src/main/java/uk/gov/hmcts/ccd/data/definition/DefaultCaseDefinitionRepository.java

```diff
@@ -112,28 +107,43 @@ public UserRole getUserRoleClassifications(String userRole) {
         } catch (Exception e) {
             if (e instanceof HttpClientErrorException
                 && ((HttpClientErrorException)e).getRawStatusCode() == RESOURCE_NOT_FOUND) {
-                LOG.debug("No classification found for user role " + userRole + " because of ", e);
+                LOG.debug("No classification found for user role {} because of ", userRole, e);
                 return null;
             } else {
-                LOG.warn("Error while retrieving classification for user role " + userRole + " because of ", e);
+                LOG.warn("Error while retrieving classification for user role {} because of ", userRole, e);
                 throw new ServiceException("Error while retrieving classification for user role " + userRole + " because of " + e.getMessage());
             }
         }
     }
 
+    @Override
+    public List<UserRole> getClassificationsForUserRoleList(List<String> userRoles) {
+        try {
+            final HttpEntity requestEntity = new HttpEntity<CaseType>(securityUtils.authorizationHeaders());
+            final Map<String, String> queryParams = new HashMap<>();
+            Optional<String> roles = userRoles.stream().reduce((s1, s2) -> s1 + "," + s2);
```

Comment:
perhaps StringUtils.join(slist, ',') is simpler?

Reply:
yes 👍